projects/storefrontlib/shared/services/import-export/export.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { FileOptions } from '@spartacus/storefront';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class ExportService {
```

Comment:
You can _export_ many various things (not only files) 😉 . 
Can we make it more specific (that it's about exporting _files_)? i.e. `ExportFileService`

Btw. I'd also rename `FileOptions` to be more related to this service `ExportFileOptions`.